clients/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java

```diff
@@ -28,13 +28,17 @@
   /**
    * Config property name ({@value #CONF_NESSIE_USERNAME}) for the user name used for (basic)
    * authentication.
+   *
+   * @deprecated "basic" HTTP authentication is not considered secure.
    */
-  public static final String CONF_NESSIE_USERNAME = "nessie.username";
+  @Deprecated public static final String CONF_NESSIE_USERNAME = "nessie.authentication.username";
```

Comment:
renaming the underlying property name and having a fallback to the old one makes sense to me, but what's the reason for deprecating the `CONF_NESSIE_USERNAME` field itself?

Reply:
production Nessie servers do not support username/password (aka "basic") authentication.

Reply:
"Basic" auth works only in "dev" and "test" environments

Reply:
ok makes sense, thanks